src/backend/base/langflow/components/processing/parse_dataframe.py

```diff
@@ -11,6 +11,7 @@ class ParseDataFrameComponent(Component):
     )
     icon = "braces"
     name = "ParseDataFrame"
+    legacy = True
```

Comment:
Maybe indicate in the description that it is deprecated and which is the replacement ?